get-did/index.html

```diff
@@ -233,6 +234,13 @@ <h4 class="footerHeading">Checkout Service:</h4>
     })
 
     const { userAgent } = navigator
+
+    const isMobile = /Android|iPhone|iPad/.test(userAgent)
+
+    if(isMobile) {
+      document.querySelector('.browsers').remove()  
+    }
```

Comment:
Would it be nicer with some formatting?
```suggestion
    const isMobile = /Android|iPhone|iPad/.test(userAgent)
    if (isMobile) {
      document.querySelector('.browsers').remove()  
    }
```